File: pclassic/2016s/PClassic2016Stubs/stubs/LightsaberLength.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileReader;
import java.io.IOException;
import java.util.Scanner;

public class LightsaberLength {

	public static void main(String[] args) throws FileNotFoundException {
		
		 Scanner fileInput = new Scanner(new File("LightsaberLengthIN.txt"));
         
	        while(fileInput.hasNext()) {
	 
	            int power = fileInput.nextInt();
	            int radius = fileInput.nextInt();
	             
	            System.out.println(measure(power, radius));
	             
	        }
	        fileInput.close();
	}
  //Fill out the body of this method
  public static int measure(int power, int radius) {
    int length = 0;
    return length;
  }
}

File: pclassic/2016s/PClassic2016Stubs/stubs/LightsaberLength.py
def measure(power, radius):
    return 0

def parse_file_and_call_function():
    with open("LightsaberLengthIN.txt", "r") as f:
        array = f.readline().split(' ')

        for x in range(0, len(array)//2):
            power = int(array[2 * x])
            radius = int(array[2 * x + 1])
            print (measure(power,radius))



if __name__ == "__main__":
    parse_file_and_call_function()
